waspc/data/Generator/templates/react-app/src/auth/useAuth.ts

```diff
@@ -1,12 +1,14 @@
 import { useQuery } from '../queries'
 import api, { handleApiError } from '../api'
 import { HttpMethod } from '../types'
+// todo(filip): turn into a proper import
+import { type SanitizedUser } from '../../../server/src/_types/' 
```

Comment:
You don't really _need_ to use the new import style, it only helps you to define a JS import statement in Haskell and then print it in a JS file.

If want to ensure that this import is importing from the proper place (server/src) you can Strong Path it and define a new `JSImport` and then output it in the template. This would move the information about this weird import in Haskell, which would be a win for us IMHO.

Reply:
Nit: remove trailing`/`